src/frontend/context/AppContext.tsx

```diff
@@ -57,6 +56,26 @@ interface AuthProviderExports {
     innloggetSaksbehandler: ISaksbehandler | undefined;
 }
 
+const tilgangModal = (data: IRestTilgang, lukkModal: () => void) => ({
+    tittel: 'Diskresjonskode',
+    lukkKnapp: true,
+    visModal: !data.saksbehandlerHarTilgang,
+    onClose: () => lukkModal(),
+    innhold: () => {
+        return (
+            <Normaltekst>
```

Comment:
Kan du endre til å bruke `BodyShort` fra `@navikt/ds-react`?

Reply:
Ah, ser at dette er flyttet opp fra senere i filen, så det vil uansett bli en merge conflict for designoppgraderingsbranchen. Da kan du bare la det være, så fikser vi det heller der :)

Reply:
Nå har jeg allerede pushet endringen :)